flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/handlers/TaskManagerLogHandler.java

```diff
@@ -141,6 +144,15 @@ public TaskManagerLogHandler(
 		timeTimeout = Time.milliseconds(timeout.toMillis());
 	}
 
+	@Override
+	public String[] getPaths() {
+		if (serveLogFile) {
```

Comment:
I think the proper way to do this would be to have an abstract `TaskManagerLogHandler` and inherit it once for the logs and once for stdout.